package.json

```
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "description": "Extensible JavaScript file upload widget with support for drag&drop, resumable uploads, previews, restrictions, file processing/encoding, remote providers like Instagram, Dropbox, Google Drive, S3 and more :dog:",
   "lint-staged": {
-    "*.js": "eslint",
+    "*.js": "eslint --fix",
```

Unrelated to unsplash, and will cause problems in the short term. We can reconsider once we've moved all packages to ESM.
```suggestion
    "*.js": "eslint",
```